### ISSUE TITLE
API Download Feedback 

### ISSUE THREAD

**Issue by Sonsalt** [SIXNetworks/withSIX-Platform#113]
**Update Three:**

After Unsubscribing all the mods via Steamworks and Restarting the Alpha Client in Admin mod, the Download of the collections appears to be working again. 

It even shows the Processing percentage and download speed, though that one might be wrong since I can not download with more than 400kb/s even though it says 2.7mb.

Therfore it appears that the subscriptions are interfering with the client: https://gyazo.com/2b85a210b036f5a3ba7cc4d18e15ae98

The Download fails at around 37% of the collection with 729426722 file missing... The client also appears to shut down or crash shortly after. 

http://staging.withsix.com/p/Starbound/collections/Nsf6gpVdnkaOESUq219H8Q/My-First-Collection
 
------------------------------

LOGS: https://mega.nz/#!XkQGRbqJ!1i6EMG43Kp4KqGDdYc7obP7Vf7Xu5oagPLz2uhTt3uw

**Update Two:**

It appears that the initial install did install the mods, however when opening the folder it shows in a different one than Sync does, plus Sync showed the mods not installed, while in-game they show up. 

After restarting the PC and the clients I am able to download Skyrim mods, but when switching to Starbound and restarting the client I again getting the errors again. 

I can not get the download to work anymore for Starbound

-----------------------

**Update One**

Since the issue started, even after restarting the PC and clients no more downloads work. Upon clicking on "install" the game shows running in steam but it actually is not. 

----------------------------------------------------
I was downloading a newly created collection: http://staging.withsix.com/p/Starbound/collections/Nsf6gpVdnkaOESUq219H8Q/My-First-Collection

At about 37% the download did not continue. After a while I closed the program and restarted it. 
Since I get this Error message:

```
An error has occured while trying to 'Action: InstallCollection'

An unexpected error has occurred while trying to execute the requested action:
C:\Program Files (x86)\Steam\steamapps\workshop\content\211820\736081761

We've been notified about the problem.

Please make sure you are running the latest version of the software.

If the problem persists, please contact Support: community.withsix.com
```

This error occeurs also when downloading any other mod for Starbound. 

In addition I can not close Steam as it keeps telling me that I first have to close starbound. 

https://gyazo.com/bdc22e843db84884b49fd53488985b3b

**Comment by sickboy** [comment]
@Sonsalt I would recommend posting a comment with back-story, and updating the first post only with facts, preferably in a checklist.

If you got the error about a mod folder, then you didn't properly reset that mod by resubscribing etc.
Anyway, the issue should be addressed by #119

**Comment by Sonsalt** [comment]
@sickboy 

Will do next time, I remember that last time I did this, you asked me to add it to the top.... 

It is hard to find the right balance..... I will try to create a digest of findings in the descriptions and keep the story in the comments..
